public/_headers

```diff
@@ -1,7 +1,9 @@
 /*
   X-Frame-Options: DENY
   X-XSS-Protection: 1; mode=block
-  Content-Security-Policy: frame-ancestors 'self'
+  Content-Security-Policy: frame-ancestors 'self'; default-src 'self'; script-src 'self'
```

Comment:
did you consider https://github.com/luniehq/lunie/blob/develop/vue.config.js#L55 ?

Reply:
I see, it is really well configured there... But checking app.lunie.io in Network I only get the header present in `headers`:

<img width="340px" src="https://user-images.githubusercontent.com/40721795/79685155-bed9a180-8236-11ea-80a5-e5820dbd1243.png">

Also, I don't understand, why don't we put all those options directly in `headers`? Why do we need to do it through webpack?

Maybe it is not working

Reply:
let'S move it all over to the netlify config then